.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+# This CircleCI workflow builds our documentation into HTML pages.
+# The generated pages are viewable as artifacts in CircleCI.
+# CircleCI can be run on PRs from forks & allows viewing stored artifacts
+# without download.
+
+version: 2.0
+
+jobs:
+  build-doc:
+    docker:
+      - image: circleci/python:3.7.3-stretch
+
+    working_directory: ~/tmp-fairlearn
+    steps:
+      # check out PR branch
+      - checkout
+
+      - run:
+          name: Upgrade pip, setuptools, and wheel before installing other dependencies
+          command: sudo python -m pip install --upgrade pip setuptools wheel
+      
+      - run:
+          name: 'Install required packages'
+          command: sudo pip install -r requirements.txt
+      
+      - run:
+          name: 'Build'
+          command: python -m sphinx -b html -j auto docs docs/_build/html
+
+      - store_artifacts:
+          path: docs/_build/html
```

Comment:
So are these visible now? If so, what's the URL?

Reply:
https://11-133444044-gh.circle-artifacts.com/0/docs/_build/html/index.html

I'd like it to be a single click from the PR view, so there's probably some extra change coming once I figure out how to add that link directly into the PR the way scikit-learn does. Right now it's a handful of clicks starting with navigating to CircleCI (see "build-webpage" in the checks and then search for the artifacts, find the one with index.html and click on it).

The link at the top should make it clear what this is for, though, I hope :-)